Client/Frontend/Widgets/BlurTableViewCell.swift

```diff
@@ -0,0 +1,98 @@
+    static let LabelColor = UIConstants.SystemBlueColor
+    static let BorderWidth: CGFloat = CGFloat(0.5)
+    static let CellSideOffset = 20
+    static let TitleLabelOffset = 10
+    static let CellTopBottomOffset = 12
+    static let StatusIconSize = 24
+    static let SelectedOverlayColor = UIColor(white: 0.0, alpha: 0.25)
+    static let CornerRadius: CGFloat = 3
+}
+
+class BlurTableViewCell: UITableViewCell {
+    lazy var titleLabel: UILabel = {
+        let titleLabel = UILabel()
+        titleLabel.font = DynamicFontHelper.defaultHelper.DeviceFontMedium
+        titleLabel.textColor = BlurTableViewCellUX.LabelColor
+        titleLabel.textAlignment = .Left
+        titleLabel.numberOfLines = 1
+        return titleLabel
+    }()
+
+    lazy var statusIcon: UIImageView = {
+        let siteImageView = UIImageView()
+        siteImageView.contentMode = UIViewContentMode.ScaleAspectFit
+        siteImageView.clipsToBounds = true
+        siteImageView.layer.cornerRadius = BlurTableViewCellUX.CornerRadius
+        return siteImageView
+    }()
+
+    lazy var selectedOverlay: UIView = {
+        let selectedOverlay = UIView()
+        selectedOverlay.backgroundColor = BlurTableViewCellUX.SelectedOverlayColor
+        selectedOverlay.hidden = true
+        return selectedOverlay
+    }()
+
+    override var selected: Bool {
+        didSet {
+            self.selectedOverlay.hidden = !selected
+        }
+    }
+
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        layer.shouldRasterize = true
+        layer.rasterizationScale = UIScreen.mainScreen().scale
+
+        isAccessibilityElement = true
+
+        contentView.addSubview(selectedOverlay)
+        contentView.addSubview(titleLabel)
+        contentView.addSubview(statusIcon)
+
+        selectedOverlay.snp_remakeConstraints { make in
+            make.edges.equalTo(contentView)
+        }
+
+        titleLabel.snp_remakeConstraints { make in
+            make.leading.equalTo(contentView).offset(12)
+            make.trailing.equalTo(statusIcon.snp_leading)
+            make.centerY.equalTo(contentView)
+        }
+
+        statusIcon.snp_remakeConstraints { make in
+            make.size.equalTo(BlurTableViewCellUX.StatusIconSize)
+            make.trailing.equalTo(contentView).inset(12)
+            make.centerY.equalTo(contentView)
+        }
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        self.preservesSuperviewLayoutMargins = false
+        self.separatorInset = UIEdgeInsetsZero
+        self.layoutMargins = UIEdgeInsetsZero
+    }
+
+    func configureCell(label: String, imageString: String) {
+        titleLabel.text = label
+
+        let image = UIImage(named: imageString)!.imageWithRenderingMode(.AlwaysTemplate)
```

Comment:
Because it might not always be you creating this menu. It's possible someone gives you a imageString for an image that doesnt exist. Will this crash if that happens?